plugins/XmlMaterialProfile/XmlMaterialProfile.py

```diff
@@ -439,6 +439,8 @@ def deserialize(self, serialized):
         meta_data["base_file"] = self.id
         meta_data["status"] = "unknown"  # TODO: Add material verfication
 
+        global_setting_values = {}
```

Comment:
"global" in this context is "global for all the containers that get created from a single xml file", not "global for all extrudertrains".